x-pack/plugins/infra/public/pages/metrics/hosts/components/tabs/tabs.tsx

```diff
@@ -14,6 +14,7 @@ import { AlertsTabContent } from './alerts';
 
 import { AlertsTabBadge } from './alerts_tab_badge';
 import { TabIds, useTabId } from '../../hooks/use_tab_id';
+import { LogsTabContent } from './logs/logs_tab_content';
```

Comment:
We're exporting it in the `./logs/index.ts` file, so we could simply:

```suggestion
import { LogsTabContent } from './logs';
```